source/includes/_documentation.md

```diff
@@ -4853,6 +4853,185 @@ The angular Button:
+        selectedValue.has().text("Selected value: left");
+    }
+```  
+
+![Exclusive button toggle](../images/angular/exclusive_button_toggle.png)
+
+```html
+<mat-button-toggle _ngcontent-pwh-c328="" value="left" aria-label="Text align left" id="left-align-button-toggle" class="mat-button-toggle mat-focus-indicator mat-button-toggle-appearance-standard" tabindex="-1">
+    <button type="button" class="mat-button-toggle-button mat-focus-indicator" id="left-align-button-toggle-button" tabindex="0" aria-pressed="false" name="mat-button-toggle-group-7" aria-label="Text align left">
+        <div class="mat-button-toggle-label-content"><mat-icon _ngcontent-pwh-c328="" role="img" class="mat-icon notranslate material-icons mat-icon-no-color" aria-hidden="true">format_align_left</mat-icon></div>
+    </button>
+    <div class="mat-button-toggle-focus-overlay"></div>
+    <div matripple="" class="mat-ripple mat-button-toggle-ripple"></div>
+</mat-button-toggle>
+```
+
+|Method | Description | Return Type
+--- | --- | ---
+**click()** | Click the button | void
+**is()** | Assert action | TextAssert
+**has()** | Assert action | TextAssert
+**attr()** | Check whether an element has attribute of specified name and with given value | IsAssert
+**displayed()** | Check that element is displayed |	TextAssert
+**enabled()** | Check that element is enabled | TextAssert
+**disabled()** | Check that element is disabled | TextAssert
+**text()** | Check whether a text matches a pattern | IsAssert
+
+
+#### <a href="https://github.com/jdi-testing/jdi-light/blob/1959-button-toggle-tests/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ButtonToggleTest.java" target="_blank">Here you can find Button toggle tests</a>
+
+### Badge
+
+```java 
+    /**@FindBy(css = "#text-with-badge span") public static Badge badgeWithText;*/
+    @UI("#text-with-badge span")
+    public static Badge badgeWithText;
+        
+    /**@FindBy(css = "#button-with-left-badge span:not(.mat-button-wrapper)") public static Badge badgeWithButton;*/
+    @UI("#button-with-left-badge span:not(.mat-button-wrapper)")
+    public static Badge badgeWithButton;
+    
+    /**@FindBy(css = "#icon-with-badge span") public static Badge badgeWithIcon;*/
+    @UI("#icon-with-badge span")
+    public static Badge badgeWithIcon;
+
+    @Test
+    public void basicBadgeTest() {
+        badgeWithText.is().displayed();
+        badgeWithButton.is().displayed();
+        badgeWithIcon.is().displayed();
+    }
+    
+    @Test
+    public void badgeTextTest() {
+        badgeWithText.has().text("4");
+        badgeWithButton.has().text("8");
+        badgeWithIcon.has().text("15");
+    }
+   } 
+```
+#### <a href="https://material.angular.io/components/badge/overview" target="_blank">Badge overview</a>
+
+Badge is located in the following class:
+
+- __Java__: _com.epam.jdi.light.angular.elements.common.Badge_
+
+![Badge](../images/angular/badge.png)
```

Comment:
Badge image is not shown.
![image](https://user-images.githubusercontent.com/65406596/87915086-50e18800-ca7a-11ea-9870-a1e3c25b0846.png)
The reason can be in naming, in file name PNG in upper case.

Reply:
Fixed